src/widgets/text.rs

```diff
@@ -51,11 +51,11 @@ pub fn Text(props: TextProps) {
         ..
     } = props.clone();
     let font_name = font;
+    let default_style = props.clone().styles.unwrap_or_default().font.resolve();
```

Comment:
Hm, if we make `font` inherit, this will always panic (since we don't know what to resolve to until after style resolution)

Reply:
This is a good point. I wonder how CSS handles inheritance in this situation. 🤔